src/classes/netstat.class.js

```diff
@@ -89,7 +89,7 @@ class Netstat {
                     });
                 }
 
-                this.si.inetLatency("1.1.1.1", (data) => {
+                this.si.inetLatency(window.settings.pingAddr, (data) => {
```

Comment:
```suggestion
                this.si.inetLatency("1.1.1.1" || window.settings.pingAddr, (data) => {
```

Suggestion above will prevent errors if the setting isn't defined.